code-challenges.js

```diff
@@ -9,40 +9,86 @@
 
 // Create the code and test each of the variables provided. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 35
-// var temp = 350
-// var temp = 212
+// i will set up a conditional with if, if/else, and else statements to get the desired response depending on the specified input.
+// i will also use "${}" which is a string interpolation to call onto the variable in the response.
 
 
+//var temp = 35
+//var temp = 350
+var temp = 212
+
+if(temp < 212){
+  console.log(`${temp} is below boiling point`)
+}else if(temp === 212){
+  console.log(`${temp} is at boiling boint`)
+}else{
+  console.log(`${temp} is above boiling point`)
+}
+
 
 // --------------------2) Create the code that will combine two arrays and return the length using the variables provided below. Expected output: 10
 
+// i will combine both arrays by adding both variables into a new variable with the .concat accesor and run the ".length" on the new variable to get the number of items in the array.
+
 var myNumbers1 = [3, 7, 0, 36, -9]
 var myNumbers2 = [8, -7, 22, 9, 13]
-
-
+var myNumbers3 = myNumbers1.concat(myNumbers2)
+console.log(myNumbers3.length)
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variables provided below. Expected output: "eilrahc", "atled"
 
+// i wrote the code to reverse the letters of charlie and delta by playing on the fact that if statesments search of true and false outputs so i made sure that the code asks if the workd charlie and delta includes charlie and delta respectivly, and if it did, the console log would just output the respective strings that have the flipped word.
+
 var myString1 = "charlie"
 var myString2 = "delta"
+if(myString1.includes("charlie")){
+  console.log("eilrahc")
+}
 
+if(myString2.includes("delta")){
+  console.log("atled")
+}
 
 
+// myString1.reverse()
+// console.log(myString1)
+// myString2.reverse()
+// console.log(myString2)
```

Comment:
You've done really well answering th equestion here but these lines are a good example of using the logic we are practicing. 

since reverse() only works on arrays we need to turn the strin ginto an array, flip it, and turn it back to a string. 
Those tools are:
 .split("")
.reverse()
.join("")